tests/_utils/classictesteditor.js

```diff
@@ -54,7 +54,7 @@ export default class ClassicTestEditor extends Editor {
 	/**
 	 * @inheritDoc
 	 */
-	static create( element, config ) {
+	static create( element, config = {} ) {
```

Comment:
We can do it more similar to the classic editor and allow also pass data as the first parameter.